packages/gatsby-plugin-offline/src/gatsby-node.js

```diff
@@ -45,13 +45,15 @@ exports.onPostBuild = (args, pluginOptions) => {
     `component---node-modules-gatsby-plugin-offline-app-shell-js`,
   ])
 
+  const omitPrefix = path => args.pathPrefix ? path.slice(args.pathPrefix.length) : path
+
   const criticalFilePaths = _.uniq(
```

Comment:
Is there a reason to not do this .map one time, instead of on each concat call?

i.e. I'd re-write like:

```js
const criticalFilePaths = _.uniq(
  _.concat(
    getResourcesFromHTML(`${process.cwd()}/${rootDir}/index.html`),
    getResourcesFromHTML(`${process.cwd()}/${rootDir}/404.html`),
    getResourcesFromHTML(
      `${process.cwd()}/${rootDir}/offline-plugin-app-shell-fallback/index.html`
    )
  )
).map(omitPrefix)
```

Reply:
ah hahah brain fart

Reply:
Happens to all of us :)